src/v3/src/transformer/i18n/transform.ts

```diff
@@ -103,8 +103,38 @@ export const transformAdditionalPhoneUITranslations = (formBag: FormBag) => {
   return formBag;
 };
 
+export const transformPasswordTooltipTranslations = (formBag: FormBag) => {
+  const { uischema } = formBag;
+
+  traverseLayout({
+    layout: uischema,
+    predicate: (element) => (element as FieldElement).options?.inputMeta?.secret === true,
+    callback: (element) => {
+      // eslint-disable-next-line no-param-reassign
+      (element as FieldElement).options = {
+        ...(element as FieldElement).options,
+        translations: [
```

Comment:
use the translations field at element root level, the one in options should be deprecated